pyam/core.py

```diff
@@ -406,12 +406,16 @@ def validate(self, criteria={}, exclude=False):
 
             return df
 
-    def filter(self, filters, keep=True, inplace=False):
+    def filter(self, keep=True, inplace=False, **kwargs):
```

Comment:
We need to decide if we want to make this API change now. How many users do we have right now? The safest way is to do the following:

- make the argument `filters=None`
- add the following check at the beginning of the function:

```
if filters is not None:
    warnings.warn('filters keyword argument in filters() is deprecated and will be removed in the next release')
    kwargs.update(filters)  
```